### ISSUE TITLE
Water coil nodes are not updated with zero chiller condenser flow 

### ISSUE THREAD

**Issue by lgu1234** [NREL/EnergyPlus#4931]
#4453 corrected node conditions of Chiller:Electric:EIR with zero condenser loop flow. However, node values of a water coil are not updated correctly. Please use a test file of #4453.

A possible reason is that when chiller nodes are updated, the program may not have a chance back to the air loop to update coil nodes.

**Comment by Myoldmopar** [comment]
So, when I run the defect file, and report out the water coil heat transfer rate, pump mass flow rate, chiller mass flow rate, and cooling coil node mass flow rates, I see:

![image](https://cloud.githubusercontent.com/assets/849824/11671623/e17a57d8-9dc6-11e5-9c4a-de94b107794f.png)

which shows that during the design day (which is 7/21), that:
- the cooling coil is providing cooling to the space
- the chiller evaporator is not providing any cooling
- the ChW supply pump _is_ on
- the chiller has no flow rate
- the cooling coil does have flow rate

I need to look at the configuration of this plant loop, but right now I'm guessing there is some other piece of cooling equipment providing cooling for the loop.  So the chiller _does_ go off, but the pump stays on to provide flow for the cooling coil.  In which case, the cooling coil _should_ still be on.  I'll look at it a little more, but maybe @lgu1234 or @mjwitte could provide a little more background of what I should be seeing/fixing.

**Comment by Myoldmopar** [comment]
Hmm, still investigating.  Against my initial thought, it would appear there _isn't_ a secondary cooling component on the chilled water loop.  And after some additional reporting, the loop temperature isn't going up.  I'll go through and look at many more temperatures and flow rates through the system of loops to figure out what is going on.

**Comment by lgu1234** [comment]
@Myoldmopar My observation is that demand side nodes are not updated, when supply side nodes are updated with no cooling available due to zero flow at the condenser.

**Comment by Myoldmopar** [comment]
OK thanks @lgu1234.  I'll let you know what I find.

**Comment by Myoldmopar** [comment]
OK, I am able to reproduce a problem.  I ran the defect with Raleigh TMY3 as specified in the idf. I am looking specifically at the `7/21 12:00` design day data point.
- As you mentioned, there is no condenser flow.  The chiller has zero flow on both the condenser and chilled water sides.
- The chilled water plant does still have flow.  The supply side flow goes through the bypass, avoiding the "unavailable" chiller.
- On the demand side, the flow goes through the chilled water coil.  And here's where it gets fun...the water temperature goes **down** through the coil.  20.73 to 20.68.  :+1: 
- Looking at the chilled water coil air-side nodes, I see the temperature of the air **also** going down...18.12 to 15.13.  :+1:

So this is something seriously wrong.  I am guessing a competition issue between plant and air side again.  

@wfbuhl I know you mentioned the issue yesterday regarding plant and air solvers, so I thought I'd tag you over here in case you have info for the fun time.

And of course, FYI: @EnergyArchmage @mjwitte

**Comment by Myoldmopar** [comment]
To add some misc notes:
- the code never hits the _splitter/mixer continuity resimulation block_ [here](https://github.com/NREL/EnergyPlus/blob/9da112c653f018f8484435fada1ff1cf68aa6593/src/EnergyPlus/HVACManager.cc#L796)
- the loops seem to converge _very_ quickly, with only two HVAC iterations every time I stepped through

More notes to come...

**Comment by Myoldmopar** [comment]
Of course, when I take a random example file to check for an energy balance on another Coil:Cooling:Water object instance, I find 2ZoneDataCenterHVAC_wEconomizer.idf, and an annual run shows the water coil (at least the one I picked first) not running, ever, through the annual run.  On to try another...

**Comment by Myoldmopar** [comment]
So `5ZoneAirCooled`.  Checking the energy balance on the`Coil:Cooling:Water` named `Main Cooling Coil 1`.  I reported the temperature/flow rate on the air and water nodes of the coil.  Then calculated the difference in heat transfer.  Scroll to the right to see all the columns (or use a browser extension to coerce GitHub to full screen width).

| Date/Time | MIXED AIR NODE 1:System Node Temperature [C](Hourly) | MIXED AIR NODE 1:System Node Mass Flow Rate [kg/s](Hourly) | MAIN COOLING COIL 1 OUTLET NODE:System Node Temperature [C](Hourly) | MAIN COOLING COIL 1 WATER INLET NODE:System Node Temperature [C](Hourly) | MAIN COOLING COIL 1 WATER INLET NODE:System Node Mass Flow Rate [kg/s](Hourly) | MAIN COOLING COIL 1 WATER OUTLET NODE:System Node Temperature [C](Hourly) | Qair [kW] | Qwater [kW] | Difference [kW](+ for higher water heat transfer) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 7/21/15 01:00 | 22.39 | 0.37 | 12.00 | 7.22 | 0.23 | 11.24 | 3.87 | 3.91 | 0.04 |
| 7/21/15 02:00 | 21.88 | 0.37 | 12.00 | 7.22 | 0.22 | 11.25 | 3.68 | 3.72 | 0.04 |
| 7/21/15 03:00 | 21.46 | 0.37 | 12.00 | 7.22 | 0.21 | 11.26 | 3.52 | 3.56 | 0.04 |
| 7/21/15 04:00 | 21.11 | 0.37 | 12.00 | 7.22 | 0.20 | 11.27 | 3.39 | 3.43 | 0.04 |
| 7/21/15 05:00 | 20.83 | 0.37 | 12.00 | 7.22 | 0.20 | 11.29 | 3.29 | 3.32 | 0.04 |
| 7/21/15 06:00 | 20.70 | 0.37 | 12.00 | 7.22 | 0.19 | 11.30 | 3.24 | 3.27 | 0.04 |
| 7/21/15 07:00 | 21.02 | 0.37 | 12.00 | 7.22 | 0.20 | 11.29 | 3.35 | 3.39 | 0.04 |
| 7/21/15 08:00 | 21.52 | 0.37 | 12.00 | 7.22 | 0.21 | 11.28 | 3.57 | 3.61 | 0.04 |
| 7/21/15 09:00 | 23.03 | 0.54 | 12.25 | 7.22 | 0.40 | 11.01 | 5.86 | 6.36 | 0.50 |
| 7/21/15 10:00 | 24.44 | 0.77 | 12.57 | 7.22 | 0.73 | 10.74 | 9.13 | 10.70 | 1.57 |
| 7/21/15 11:00 | 25.40 | 0.95 | 13.44 | 7.22 | 0.79 | 11.28 | 11.38 | 13.33 | 1.95 |
| 7/21/15 12:00 | 26.04 | 0.94 | 13.67 | 7.22 | 0.79 | 11.45 | 11.65 | 13.90 | 2.25 |
| 7/21/15 13:00 | 26.63 | 0.76 | 13.09 | 7.22 | 0.79 | 11.03 | 10.32 | 12.50 | 2.19 |
| 7/21/15 14:00 | 27.07 | 0.96 | 13.85 | 7.22 | 0.79 | 11.53 | 12.79 | 14.13 | 1.34 |
| 7/21/15 15:00 | 27.28 | 1.07 | 14.38 | 7.22 | 0.79 | 11.92 | 13.82 | 15.42 | 1.59 |
| 7/21/15 16:00 | 27.44 | 1.12 | 14.76 | 7.22 | 0.79 | 12.21 | 14.32 | 16.37 | 2.05 |
| 7/21/15 17:00 | 27.48 | 1.10 | 14.77 | 7.22 | 0.79 | 12.23 | 14.02 | 16.46 | 2.44 |
| 7/21/15 18:00 | 27.43 | 1.01 | 14.49 | 7.22 | 0.79 | 12.04 | 13.13 | 15.82 | 2.70 |
| 7/21/15 19:00 | 27.39 | 0.48 | 12.28 | 7.22 | 0.67 | 10.64 | 7.33 | 9.61 | 2.28 |
| 7/21/15 20:00 | 26.77 | 0.37 | 12.00 | 7.22 | 0.46 | 10.79 | 5.50 | 6.83 | 1.34 |
| 7/21/15 21:00 | 25.74 | 0.37 | 12.00 | 7.22 | 0.37 | 10.93 | 5.11 | 5.81 | 0.70 |
| 7/21/15 22:00 | 24.64 | 0.37 | 12.00 | 7.22 | 0.32 | 11.05 | 4.70 | 5.07 | 0.37 |
| 7/21/15 23:00 | 23.75 | 0.37 | 12.00 | 7.22 | 0.28 | 11.12 | 4.37 | 4.56 | 0.19 |
| 7/22/15 00:00 | 23.01 | 0.37 | 12.00 | 7.22 | 0.25 | 11.20 | 4.10 | 4.17 | 0.08 |

Ummm.  Am I missing something?  I feel like I am missing something.  Even though I am directly reporting the node conditions on the simple water to coil, I was ready for some noise throughout the day, but I didn't expect a strong bias in one direction, on the order of 2.5 kW difference, or about 15%....  

@mjwitte @rraustad @EnergyArchmage @wfbuhl

**Comment by Myoldmopar** [comment]
Ha, the output in the csv header like:

```
TLET NODE:System Node Temperature [C](Hourly)
```

is interpreting the `[C](Hourly)` as a hyperlink.

**Comment by mjwitte** [comment]
As another check, what is the coil reporting for Cooling Coil Total Cooling Rate?

**Comment by Myoldmopar** [comment]
No point in beating around the bush, that was dumb.  I should probably have included latent when comparing it against the waterside heat transfer rate.  Gah.  OK, something is still up with the defect, but at least I will be able to breath...
